app/models/post.rb

```diff
@@ -1,5 +1,6 @@
 class Post < ApplicationRecord
   belongs_to :user
+  has_many :comments
   validates :title, presence: true, length: { in: 20..80 }, uniqueness: true
-  validates :body, presence: true, length: { in: 200..10000}
+  validates :body, presence: true, length: { in: 200..10_000 }
```

Comment:
### [OPTIONAL]

- [ ] You can make the range for the length smaller so it is easier to test without having to add of 20 characters for a title and 200 for the body.